app/controllers/concerns/attachment_manageable.rb

```diff
@@ -5,4 +5,15 @@ def allow_only_one_attachment
     flash.now[:alert] = t(".attachment_manageable.upload_limit_exceeded")
     render turbo_stream: turbo_stream.replace("flash", partial: "layouts/shared/flash_messages")
   end
+  
+  def alert_for_no_images
+    if params[:action] === 'attach_images' and params[:pet][:images].length > 1
```

Comment:
`==` in Ruby if you want equality
https://dev.to/baweaver/understanding-ruby-triple-equals-2p9c `===` in Ruby means something else